tasks/util/patch_minified.js

```diff
@@ -11,5 +12,20 @@ var STR_NEW = 'require("+ a(r) +");';
  *
  */
 module.exports = function patchMinified(minifiedCode) {
-    return minifiedCode.replace(STR_TO_REPLACE, STR_NEW);
+    for(var i = 0; i < ALPHABET.length; i++) {
+        var li = ALPHABET[i];
+
+        for(var j = 0; j < ALPHABET.length; j++) {
+            var lj = ALPHABET[j];
+
+            var MIDDLE = li + '(' + lj + ')';
+
+            var strOld = FRONT + MIDDLE + BACK,
+                strNew = FRONT + ' ' + MIDDLE + ' ' + BACK;
+
+            minifiedCode = minifiedCode.replace(strOld, strNew);
```

Comment:
It could also be done using `.split(strOld).join(strNew)` but I have no idea if it would be any faster in this case.